modules/standard/Path.chpl

```diff
@@ -434,56 +434,58 @@ proc file.realPath(): string throws {
    :rtype: `string`
   */
 
-  proc normPath(name: string): string {
+  //removes ".." from given string
+  proc remove(inout result: string): void {
```

Comment:
Having the helper function first will disassociate the documentation comment from normPath itself.  Please move the comment to attach it to the proper function.

Also, you'll want to mark the remove function as `pragma 'no doc';` or `private` (probably `private`), as I don't expect you want people to use it outside of this context